shell.nix

```diff
@@ -19,7 +19,7 @@ let
   #
   # > tools = {
   # >   ...
-  # >   haskell-language-server = "1.1.0";
+  # >   haskell-language-server = "1.2.0";
```

Comment:
1.2.0 or latest?

Reply:
Should have been "1.2.0.0" but it was complaining :smile:

Reply:
Fixed